<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GameListView</class>
 <widget class="QWidget" name="GameListView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>479</width>
    <height>368</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Add / Edit Game</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="1" colspan="2">
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="2" column="0" colspan="3">
      <widget class="QLabel" name="platformLabel">
       <property name="text">
        <string>Platform:</string>
       </property>
      </widget>
     </item>
     <item row="8" column="2">
      <widget class="QPushButton" name="deleteButton">
       <property name="text">
        <string>Delete</string>
       </property>
      </widget>
     </item>
     <item row="5" column="0" colspan="3">
      <widget class="QLineEdit" name="ratingEdit"/>
     </item>
     <item row="8" column="0">
      <widget class="QPushButton" name="saveButton">
       <property name="text">
        <string>Save</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0" colspan="3">
      <widget class="QLabel" name="nameLabel">
       <property name="text">
        <string>Name:</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0" colspan="3">
      <widget class="QLineEdit" name="platformEdit"/>
     </item>
     <item row="4" column="0" colspan="3">
      <widget class="QLabel" name="ratingLabel">
       <property name="text">
        <string>Rating:</string>
       </property>
      </widget>
     </item>
     <item row="8" column="3">
      <widget class="QPushButton" name="cancelButton">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
     <item row="7" column="0" colspan="4">
      <widget class="QTextEdit" name="reviewEdit">
       <property name="styleSheet">
        <string notr="true"/>
       </property>
      </widget>
     </item>
     <item row="1" column="3" rowspan="5">
      <widget class="QLabel" name="imageLabel">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Ignored" vsizetype="Ignored">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="6" column="0" colspan="3">
      <widget class="QLabel" name="reviewLabel">
       <property name="text">
        <string>Review:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0" colspan="3">
      <widget class="QLineEdit" name="nameEdit"/>
     </item>
     <item row="8" column="1">
      <widget class="QPushButton" name="moveButton">
       <property name="text">
        <string>Move To...</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>nameEdit</tabstop>
  <tabstop>platformEdit</tabstop>
  <tabstop>ratingEdit</tabstop>
  <tabstop>reviewEdit</tabstop>
  <tabstop>saveButton</tabstop>
  <tabstop>moveButton</tabstop>
  <tabstop>deleteButton</tabstop>
  <tabstop>cancelButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
